app/scss/_footer.scss

```diff
@@ -0,0 +1,7 @@
+.footer {
+  grid-area: footer;
+  padding: 0.25em;
```

Comment:
While the footer height is auto your solution is ok. But to make it more flexible on different sizes, better set it `display: flex;` and position items in it with "justify" and "align" properties